drake/systems/plants/constraint/MultipleTimeKinematicConstraint.m

```diff
@@ -1,24 +1,17 @@
 classdef MultipleTimeKinematicConstraint < RigidBodyConstraint
   % A abstract class, that its eval function takes multiple time points as
   % the input, instead of being evluated at a single time.
-    
+
   methods
-    function obj = MultipleTimeKinematicConstraint(robot,tspan)
-      obj = obj@RigidBodyConstraint(RigidBodyConstraint.MultipleTimeKinematicConstraintCategory,robot,tspan);
+    function obj = MultipleTimeKinematicConstraint(robot)
+      obj = obj@RigidBodyConstraint(RigidBodyConstraint.MultipleTimeKinematicConstraintCategory,robot);
     end
-    
+
     function flag = isTimeValid(obj,t)
-      n_breaks = size(t,2);
-      if(n_breaks <=1)
-        error('Drake:WorldFixedPositionConstraint: t must have more than 1 entry');
-      end
-      flag = t>=obj.tspan(1)&t<=obj.tspan(end);
-    end
-    
-    function tspan = getTspan(obj)
-      tspan = obj.tspan;
+      tspan = [-inf inf];
```

Comment:
We should remove this function, as it always returns true. I imagine the way we use `MultipleTimeKinematicConstraint`, is that we create a constraint, for example (WorldFixedBodyPoseConstraint), and then add it to the solver by saying this constraint is active at some knot point (for example, if we want to say it is always active, then the knot point is 1:obj.N in DirectTrajectoryOptimization). We do not need the constraint itself to determine whether it should be active or not; the activeness is specified when we add it to the optimization problem.